### PR TITLE
Add NEWS.md

### PR DESCRIPTION
**Pull request by speidy**

### CODE REVIEW

**Comment by metalefty** [comment]
LGTM.

**Comment by proski** [comment]
Changes to the submodules should not be merged!

Please add NEWS.md to EXTRA_DIST in the top-level Makefile.am to have it distributed.

**Comment by speidy** [comment]
Fixed.

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Release notes for xrdp v0.9.1 (2016/12/21)
+## Added
+  * New xorgxrdp backend using existing Xorg with additional modules
+  * Improvements to X11rdp backend
+  * Support for IPv6 (disabled by default)
+  * Initial support for RemoteFX Codec (disabled by default)
+  * Support for multiple security layers (RDP and TLS).
```
Comment (proski) [inline NEWS.md]: No period at the end (for consistency), here and elsewhere

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  * Support for disabling deprecated ssl versions (like SSLv3), and also select custom cipher suites in xrdp.ini
+  * Support for bi-directional fastpath
+  * Support clients that doesn't support drawing orders, such as MS RDP client for Android, ChromeRDP (disabled by default)
```
Comment (proski) [inline NEWS.md]: doesn't -> don't

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  * More configurable login screen
+  * Support for several new channels:
+      * rdpdr: Device Redirection
+      * rdpsnd: Audio Output
+      * cliprdr: Clipboard
+      * xrdpvr: xrdp video redirection channel (can be used along with NeutrinoRDP client)
+  * Support for disabling channels globally or by session type
+  * Allow to specify the path for backends (Xorg, X11rdp, Xvnc)
+  * Added files for systemd support
+  * Multi monitor support
```
Comment (proski) [inline NEWS.md]: "Multi monitor" -> multi-monitor

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Changed
+* Transport: xrdp will prefer TLS transport when possible by client.
+* RDP: xrdp will do fastpath commands by default now.
+* xrdp-chansrv: chansrv logs are stored in `${XDG_DATA_HOME}/xrdp` now
```
Comment (proski) [inline NEWS.md]: Please remove "chansrv", it's implied by "xrdp-chansrv". The whole section should have a space before "*" for consistency with other sections.

File: NEWS.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      * cliprdr: Clipboard
+      * xrdpvr: xrdp video redirection channel (can be used along with NeutrinoRDP client)
+  * Support for disabling channels globally or by session type
+  * Allow to specify the path for backends (Xorg, X11rdp, Xvnc)
+  * Added files for systemd support
+  * Multi monitor support
+
+## Changed
+* Transport: xrdp will prefer TLS transport when possible by client.
+* RDP: xrdp will do fastpath commands by default now.
```
Comment (proski) [inline NEWS.md]: Do we really need to list security layers and fastpath in two sections?

**Comment by proski** [comment]
I've updated https://github.com/neutrinolabs/xrdp/wiki/NEWS

Let's use the wiki as the master version. Wiki is better suited for the editorial work.

Speaking of Makefile.am, I would prefer to have EXTRA_DIST entries sorted in POSIX locale (NEWS.md after COPYING), but it's nitpicking.

File: NEWS.md (hunk 5 above)
Comment (speidy) [inline NEWS.md]: yea. I don't see a relation between them.

**Comment by speidy** [comment]
ok, fixed now.

**Pull request merged**